src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -213,6 +234,12 @@ export default defineComponent({
         ? this.onboardingData.entranceYear
         : placeholderText;
     },
+    gradSemText(): string {
+      return this.onboardingData.gradSem ?? placeholderText;
```

Comment:
couldn't test the placeholder text because I'm not sure if it's even possible to reach review without having a season

Reply:
It shouldn't be! This is here just to handle the type as it might be `''`

Reply:
ok!